doc/onboarding/model/form.md

```diff
@@ -0,0 +1,30 @@
+# `form` State
+
+## Definition
+State of a form that includes its values but also :
+ - required field or
+ - ability to submit or not all the form.
+
+```
+{
+    <boolean> disabled: false
```

Comment:
Ok so it's confused here. A form can not be ```disabled```, that doesn't make sense. If the password is not enough complex and we have an error, it's just ```not validated```. All these stuffs should be handled by the view specific validating function (which is used by the ```onboarding``` component). So maybe this boolean is not necessary. What do you think?